### PR TITLE
Clarify Inventory config /w comments

### PR DESCRIPTION
**Pull request by cjnovak98**
I was talking with @sean-m-sullivan the other day and while solving some other problems I had in my configuration, he had clarified a few things, which I believe if included as comments in the inventory files, would make it easier for newbies like myself to understand how to properly configure and use the template the team has put together.

This is my first shot at improving it's clarity and understanding. I can edit the remaining inventory files to read similarly, but wanted to start a discussion first. Let me know what you think!

Also obligatory this is my first time committing to a community developed repo.

### CODE REVIEW

File: inventory_prod.yml [hunk 1]
```diff
@@ -3,23 +3,22 @@ all:
...
     prod:
       hosts:
-        127.0.0.1:
+        127.0.0.1: # Due to the way this template is configured, DO NOT CHANGE THIS!
       vars:
         connection: local
     automationcontroller:
       hosts:
-        HERE:
+        HERE: # FQDN of Automation Controller host
```
Comment (djdanielsson) [inline inventory_prod.yml]: this should say hosts not host because it could be more then one

File: inventory_prod.yml [hunk 2]
```diff
@@ -3,23 +3,22 @@ all:
...
         connection: local
     automationcontroller:
       hosts:
-        HERE:
+        HERE: # FQDN of Automation Controller host 
 
     automationhub:
       hosts:
-        HERE:
+        HERE: # FQDN of Automation Hub host. Can be commented out if not in use.
```
Comment (djdanielsson) [inline inventory_prod.yml]: this should say hosts not host because it could be more then one

**Comment by cjnovak98** [comment]
Applied the changes you requested, applied them to all of the inventory files, with exception to the inventory_ci.yml as I don't fully understand the purpose of that one. I assume it's purpose is for pipeline/testing.

**Review by djdanielsson — APPROVED** [review]
LGTM, thanks for the PR

**Comment by djdanielsson** [comment]
yes the ci is for my pipeline testing but that is fine to update just in case. I should prob try and hide it honestly

**Pull request merged**